bootstrap.d/x11-libs.yml

```diff
@@ -2027,3 +2027,36 @@ packages:
       - args: ['make', 'install']
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
+
+  - name: libxcvt
+    labels: [aarch64]
+    architecture: '@OPTION:arch@'
+    source:
+      subdir: ports
+      git: 'https://gitlab.freedesktop.org/xorg/lib/libxcvt.git'
+      tag: 'libxcvt-0.1.2'
+      version: '0.1.2'
+    metadata:
```

Comment:
Wrong location, other ports put metadata above the source declaration. For consistency's sake we should do the same here.